qemu/tests/plug_cdrom.py

```diff
@@ -44,11 +44,8 @@ def _shutdown_vm(session):
 
     def _reboot_vm(session):
         """Reboot vm."""
-        reboot_cmd = params["reboot_cmd"]
-        error_context.context(
-            "Rebooting VM by \"%s\"." % reboot_cmd, logging.info)
-        session.sendline(reboot_cmd)
-        return vm.wait_for_login(timeout=360)
+        error_context.context("Rebooting VM.", logging.info)
+        return vm.reboot(session=session, timeout=360)
```

Comment:
The `vm.reboot` will send comand `reboot_cmd = self.params.get("reboot_command")` which is `reboot_command = shutdown -r now` in linux, but this case sends the command is "reboot".
```
    variants:
        - reboot_vm:
            Windows:
                reboot_cmd = shutdown -t 0 -r
                shutdown_cmd = shutdown -t 0 -s
            Linux:
                reboot_cmd = reboot
                shutdown_cmd = shutdown -h now
```
Maybe you could update the manul case steps if it doesn't affect your testing.
Or change the `reboot_command` for this case.
CC @peixiu

Reply:
Cases updated.